main.cpp

```diff
@@ -58,14 +60,14 @@ struct Leetcode//1
     void printThingone();
 };
 
-Leetcode::Leetcode()
+Leetcode::Leetcode() : a(0)
 {
     k = 1;
 }
 
 void Leetcode::printThingone()
 {
-    std::cout << "Leetcode::printThing() " << k << std::endl; 
+    std::cout << "Leetcode::printThing()a:" << a << " b:" << w << std::endl; 
 }
```

Comment:
add spaces in your output so the output is more readable:
```
std::cout << "Leetcode::printThing()a:" << a << " b:" << w << std::endl;
```
vs
```
std::cout << "Leetcode::printThing() a: " << a << " b: " << w << std::endl;
```
Also, you're printing out `b:` , but using the member variable `w`
Make sure the text you're printing out is accurate.
- [ ] revise